### ISSUE TITLE
Attempting to save a Reddit tab causes the tab to freeze and crash.

### ISSUE THREAD

**Issue by Starfire202** [gildas-lormeau/SingleFile#1437]
Whenever I attempt to save Reddit posts (using this one), after clicking on all the spoilers, I attempt to run the extension. It manages frame contents, but at deferrered images it'll start to freeze, and at step 1/3 it will crash. This repeats every time.

Expected behavior is that it should go through normally.

This is a screen recording of me attempting to save the tab:
https://github.com/gildas-lormeau/SingleFile/assets/156129444/dbb0d2c0-b12e-4629-8d2e-5ecd70ea60c9

 - OS: MacOS Ventura 13.0
 - Browser: Chrome
 - Version: Version 123.0.6312.107 (Official Build) (arm64)

**Comment by gildas-lormeau** [comment]
I suspect it's related to https://github.com/gildas-lormeau/SingleFile/issues/1387. This might be also exacerbated by another extension you use.

**Comment by Starfire202** [comment]
I tried getting ending every intensive tab I had and some extensions (besides SingleFile and the Reddit tab of course), and surprisingly it didn't freezer, but it still crashed. I'm not sure how it would be related; my Chrome can handle large downloads pretty well (keep in mind I know nothing about coding and stuff like this). The Reddit post is this btw: https://www.reddit.com/r/Parahumans/comments/gcvvlv/pho_sunday_fey_seizes_reactors_leaving_millions/?sort=old

**Comment by kupietools** [comment]
Came here to report the same thing. Just happened with https://www.reddit.com/r/recruitinghell/comments/qhg5jo/this_resume_got_me_an_interview/ but seen it a few times in the last few days.

**Comment by Zireael07** [comment]
I'm on Vivaldi browser and experiencing the same. Can't save any Reddit post.

**Comment by kupietools** [comment]
Yeah, this is still tripping me up, months after first commenting on it.

**Comment by Starfire202** [comment]
That didn't seem to fix it.

**Comment by gildas-lormeau** [comment]
That is why the issue is not closed. Reddit's developers have gone to great lengths to make the whole thing very slow.

**Comment by gildas-lormeau** [comment]
Note however that you should be able to save some discussions, e.g. https://www.reddit.com/r/webdev/comments/1exfmwl/what_websites_do_you_visit_daily_as_a_developer/ without crashing the browser.
